crates/shared/src/bad_token/token_owner_finder.rs

```diff
@@ -75,21 +75,13 @@ pub struct Arguments {
     #[clap(long, env, default_value = "static", value_enum)]
     pub token_owner_finder_uniswap_v3_fee_values: FeeValues,
 
-    /// Override the default blockscout API url for this network
-    #[clap(long, env)]
-    pub blockscout_api_url: Option<Url>,
-
-    /// The blockscout API key.
-    #[clap(long, env)]
-    pub blockscout_api_key: Option<String>,
+    /// The blockscout configuration.
+    #[clap(flatten)]
+    pub blockscout: Blockscout,
```

Comment:
Shouldn't the groups themselves be optional and the individual fields in the struct be required?
That way you only have the `blockscout` struct if everything is configured correctly and don't have to map optionals awkwardly.

Reply:
Seems like this can be achieved by making the fields not `Option`s but setting `required=false` on the field macro.